configs/KAKUTEF4WING/config.h

```diff
@@ -39,6 +39,8 @@
 #define USE_FLASH_W25Q128FV
 #define USE_MAX7456
 
+#define USE_WING

```

Comment:
Guess this will be a cloud build option - so we need a gate here.
```suggestion
#ifndef USE_WING
#define USE_WING
#endif
```